reactions/src/HALO.cpp

```diff
@@ -671,6 +715,147 @@ double HALO::plinear_cosmosis(double k) const {
 }
 
 
+// vars:
+// 0 = acol, 1= omega0, 2 = mg param
+void HALO::react_init_nu(double vars[]) const{
+  SPT spt(C, P_l, epsrel);
+
+  double pspt, psptp, p1h, p1hp, plm, plcb, plnu, psptcbv, argument;
+
+  // fv
+  double fv  = vars[1]/vars[7];
+  double fv2 = pow(fv,2);
+  double fvt = 1.-fv;
+  double fvt2  = pow(fvt,2);
+
+/// mathcal E///
+  bigE = pow2(1.-fv)*one_halo(0.01,vars)/one_halop(0.01,vars);
+
+  /// Calculate kstar///
+// linear terms
+  plm = P_l(0.06);
+  plcb = P_cb(0.06);
+  plnu = P_nu(0.06);
+
+// 1halo terms
+  p1h = one_halo(0.06,vars); // real
+  p1hp = one_halop(0.06,vars); // pseudo
+
+// spt terms
+// Real PT spectrum
+  pspt = spt.PLOOPn2(1, vars, 0.06, 1e-3) + p1h ;
+// GR-1-loop spectrum with linear growth replaced by modified gravity growth (unscreened)
+  psptp = spt.PLOOPn2(8, vars, 0.06, 1e-3)+ p1hp ;
+
+// cbv terms
+  psptcbv = pspt*plnu;
```

Comment:
I'd expect this to be `psptcbv = sqrt(pspt*plnu)` to agree with our documentation (Equation (1.2) in the Overleaf "Halo model reaction" section), instead of putting the `sqrt` later.

Reply:
Agree. Have changed it.